go.mod

```diff
@@ -1,20 +1,22 @@
 module github.com/openshift/library-go
 
-go 1.22.0
+go 1.22.4
 
-toolchain go1.22.1
+toolchain go1.23.0
 
 require (
 	github.com/RangelReale/osincli v0.0.0-20160924135400-fababb0555f2
 	github.com/blang/semver/v4 v4.0.0
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
-	github.com/distribution/distribution/v3 v3.0.0-20230511163743-f7717b7855ca
+	github.com/distribution/distribution/v3 v3.0.0-beta.1
+	github.com/distribution/reference v0.6.0
```

Comment:
I'd prefer to update to the same version as kube 1.31: https://github.com/kubernetes/kubernetes/blob/release-1.31/go.mod#L26

Reply:
@bertinatto are you talking only about `github.com/distribution/reference v0.5.0` ?

Reply:
@aguidirh correct.